libctru/include/3ds/services/y2r.h

```diff
@@ -1,113 +1,361 @@
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetInputFormat(Y2R_InputFormat format);
+
+
+/**
+ * @brief Used to configure the output format
+ * @param format The output format to be used by the y2r engine
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetOutputFormat(Y2R_OutputFormat format);
+
+/**
+ * @brief Used to configure the rotation of the output
+ * @param rotation The rotation angle to be used by the y2r engine
+ *
+ * It seems to apply the rotation to strips of dimensions 8*height.
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetRotation(Y2R_Rotation rotation);
+
+/**
+ * @brief Used to configure the rotation of the output
+ * @param alignment The alignment to be used by the y2r engine for the output buffer
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
 Result Y2RU_SetBlockAlignment(Y2R_BlockAlignment alignment);
 
+/**
+ * @brief Used to configure the width of the image
+ * @param line_width Width of the image in pixels. Must be a multiple of 8, up to 1024.
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
+Result Y2RU_SetInputLineWidth(u16 line_width);
+
+/**
+ * @brief Used to configure the height of the image
+ * @param num_lines Number of lines to be converted.
+ *
+ * A multiple of 8 seems to be preferred.
+ * If using the @ref BLOCK_8_BY_8 mode, it must be a multiple of 8.
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
+Result Y2RU_SetInputLines(u16 num_lines);
+
+/**
+ * @brief Used to configure the color conversion formula
+ * @param coefficients A pointer to the coefficients to be used by the y2r engine
+ *
+ * See @ref Y2R_ColorCoefficients for more information about the coefficients.
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
+Result Y2RU_SetCoefficients(const Y2R_ColorCoefficients* coefficients);
+
+/**
+ * @brief Used to configure the color conversion formula with stantard coefficients
+ * @param coefficient Standard values of the coefficients to be used by the y2r engine.
+ *
+ * See @ref Y2R_ColorCoefficients for more information about the coefficients.
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
+Result Y2RU_SetStandardCoefficient(Y2R_StandardCoefficient coefficient);
+
+/**
+ * @brief Used to configure the alpha value of the output
+ * @param alpha Value to be used by the y2r engine for the output
+ *
+ * @note Prefer using @ref Y2RU_SetConversionParams if you have to set multiple parameters.
+ */
+Result Y2RU_SetAlpha(u16 alpha);
+
+/**
+ * @brief Used to configure the end of conversion event
```

Comment:
Between "configure" and "enable" I still prefer the latter, since the only latitude you have it enabling/disabling it, and nothing else.